src/dataExplorer/components/Results.tsx

```diff
@@ -142,7 +145,11 @@ const TableResults: FC<{search: string}> = ({search}) => {
 
   return (
     <div
-      className="data-explorer-results--view"
```

Comment:
Defer to you, but could probably use the `classNames` library to make this logic a little simpler.

Reply:
Good to know about this library! Updated